packages/odyssey-react/src/components/Table/TableBody.tsx

```diff
@@ -10,17 +10,16 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import type { ReactElement, ComponentProps } from "react";
+import type { ReactElement, ComponentPropsWithoutRef } from "react";
 import { forwardRef } from "react";
 import { useOmit } from "../../utils";
 
-export type Props = {
+export interface Props
+  extends Omit<ComponentPropsWithoutRef<"tbody">, "style" | "className"> {
   children?: ReactElement | ReactElement[];
-} & ComponentProps<"tbody">;
+}
 
-type Ref = HTMLTableSectionElement;
-
-const TableBody = forwardRef<Ref, Props>((props, ref) => {
+const TableBody = forwardRef<HTMLTableSectionElement, Props>((props, ref) => {
```

Comment:
lets add an explicit `displayName` for all these static components prior to exporting them to be assigned onto `Table`